Ez namey.js

```diff
@@ -0,0 +1,22 @@
+let firstWords = ["Innovative",
+"Creative",
+"Dynamic",
+"Visionary",
+"NextGen",
+"Bright",
+"Future",
+"Tech",
+"Smart",
+"Global"];
+let secondWords = ["Solutions",
+"Enterprises",
+"Technologies",
+"Innovations",
+"Concepts",
+"Ventures",
+"Systems",
+"Networks",
+"Dynamics",
+"Strategies"];
+let startupName = firstWords[4] + " " + secondWords[8]
```

Comment:
Could you try the `hint` from the instructions?
```
Hint: you can use this code to generate a random number from 0-9, if you dont want to specify the indexes yourself.
```
You would need Math.floor() and Math.random to give you the number index, then store it in a variable and used it instead of the `4` and `8` in your code. Just google "javascript get a random number between 1 and 10".